### PR TITLE
test(onboarding): fix signup with seed phrase tests

### LINKED ISSUE #6744: [Automation] Onboarding squish tests fail on Linux
Because of wrong names mostly, some of the test for onboarding fail. 

Either wait for this to be merged or branch on top https://github.com/status-im/status-desktop/pull/6735

### PR DESCRIPTION
**Pull request by jrainville**
Fixes #6744

The seed phrase tests were pretty messed up. I don't even know if they ever worked.

Anyway, they are upgraded and cleaned up.

StatusQ PR: https://github.com/status-im/StatusQ/pull/820
Branched on top of: https://github.com/status-im/status-desktop/pull/6735

### CODE REVIEW

**Comment by jrainville** [comment]
@endulab you are the tester for this PR (test the tests lol)

File: test/ui-test/src/screens/StatusWelcomeScreen.py [hunk 1]
```diff
@@ -59,17 +84,46 @@ def agree_terms_conditions_and_navigate_to_import_seed_phrase(self):
...
-
-        type(SeedPhraseComponents.SEEDS_WORDS_TEXTFIELD.value, seed)
+        elif len(words) == 18:
+            click_obj_by_name(SeedPhraseComponents.EIGHTEEN_WORDS_BUTTON.value)
+        elif len(words) == 24:
+            click_obj_by_name(SeedPhraseComponents.TWENTY_FOUR_BUTTON.value)
+        else:
+            test.fail("Wrong amount of seed words", len(words))
+
+        type(SeedPhraseComponents.SEEDS_WORDS_TEXTFIELD_1.value, words[0])
```
Comment (alaibe) [inline test/ui-test/src/screens/StatusWelcomeScreen.py:99]: maybe add a function, we do same thing in the wallet test?

**Review by alaibe — APPROVED** [review]
Yeah they never worked for me either

**Review by faust4exe — APPROVED** [review]

**Comment by jrainville** [comment]
@alaibe done. Can you give it a second look?

**Comment by endulab** [comment]
@jrainville I ran tests and there is a fail:

image

**Review by endulab — CHANGES_REQUESTED** [review]
- [x] fix failing Squish test

**Comment by jrainville** [comment]
@endulab thanks  for the test. Turns out I wasn't clear in my description, but I had fixed the signUp test and not the login test.

Anyway, I fixed the login test as well, so both should work now.

**Comment by endulab** [comment]
image

**Review by endulab — APPROVED** [review]

**Pull request merged**